classes/check/enabled.php

```diff
@@ -0,0 +1,82 @@
+<?php
+// This file is part of Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+/**
+ * Standard log enabled check
+ *
+ * @package    logstore_standardqueued
+ * @author     Dmitrii Metelkin
+ * @copyright  Catalyst IT
+ * @license    http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ */
+
+namespace logstore_standardqueued\check;
+
+use core\check\check;
+use core\check\result;
+use action_link;
+use moodle_url;
+use logstore_standardqueued\log\store;
+
+/**
+ * Standard log enabled check
+ *
+ * @package    logstore_standardqueued
+ * @author     Dmitrii Metelkin
+ * @copyright  Catalyst IT
+ * @license    http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ */
+class enabled extends check {
+
+    /**
+     * Is the check failed?
+     * @return bool
+     */
+    private function is_check_failed(): bool {
+        return store::both_logstore_standard_enabled();
+    }
+
+    /**
+     * Return action link
+     * @return action_link
```

Comment:
Return type in the phpdoc should match the actual return typehint.

Reply:
good catch. Fixed.